src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -21,27 +21,33 @@
  *******************************************************************************
  ******************************************************************************/
 
+#include "general-commissioning-server.h"
+
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app-common/zap-generated/cluster-objects.h>
 #include <app/AttributeAccessInterfaceRegistry.h>
-#include <app/CommandHandler.h>
-#include <app/ConcreteCommandPath.h>
-#include <app/server/CommissioningWindowManager.h>
+#include <app/reporting/reporting.h>
 #include <app/server/Server.h>
-#include <app/util/attribute-storage.h>
-#include <lib/support/Span.h>
+#include <lib/support/CodeUtils.h>
 #include <lib/support/logging/CHIPLogging.h>
 #include <platform/CHIPDeviceConfig.h>
 #include <platform/ConfigurationManager.h>
 #include <platform/DeviceControlServer.h>
 #include <tracing/macros.h>
+#include <transport/SecureSession.h>
+
+#if CHIP_CONFIG_TC_REQUIRED
+#include <app/server/TermsAndConditionsManager.h>
```

Comment:
if you place this in a separate build target, there should be dependencies added to that build target into `chip_data_model.gni`